apps/condo/domains/user/UserExternalIdentityMiddleware.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
```

Comment:
It's strange to put it inside the `user` folder. You need to move it to integrations folder

Reply:
moved